packages/fxa-auth-server/lib/routes/subscriptions/stripe.ts

```diff
@@ -158,38 +163,66 @@ export class StripeHandler {
         ...(pConfig.capabilities?.[ALL_RPS_CAPABILITIES_KEY] || [])
       );
 
-      const capabilityKeys = Object.keys(metadata).filter((key) =>
-        key.startsWith('capabilities:')
+      result = ClientIdCapabilityMap.merge(
+        result,
+        clientIdCapabilityMapFromMetadata(
+          plan.plan_metadata || {},
```

Comment:
Could this be replaced with the `metadata` const defined on line 158? This already merges the product and plan metadata, and then it's not necessary to run `clientIdCapabilityMapFromMetadata` on `plan.plan_metadata` and `plan.product_metadata`?

Reply:
Thanks for the clarification! I updated it to use only `metadata`.